src/webextension/list/components/item-summary.js

```diff
@@ -13,13 +13,15 @@ export default function ItemSummary({id, title, username}) {
   title = title.trim();
   username = username.trim();
 
-  const titleId = `item-summary-${id === NEW_ITEM_ID ? "new-title" : "title"}`;
+  const idPrefix = `${id === NEW_ITEM_ID ? "new-" : ""}`;
```

Comment:
You don't need to do string interpolation here; `const idPrefix = id === NEW_ITEM_ID ? "new-" : "";` is sufficient.

Nit: Also, this isn't actually a prefix, so maybe something like `idModifier`? (I won't cry too much if you leave it as-is though.)

Reply:
>You don't need to do string interpolation here

Oh! Right. I was in copy/paste mode 🤦‍♂️ 

>this isn't actually a prefix

Yeah I wasn't happy with it but figured "close enough" Modifier is better. 👍